### PR TITLE
templates: show cataloger tools to catalogers only

### PR DESCRIPTION
**Pull request by jmartinm**
Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
* [ ] Create test for `is_cataloger` filter.

**Comment by jmartinm** [comment]
Ready for review.

File: tests/unit/theme/test_theme_jinja2filters.py [hunk 1]
```diff
@@ -1227,3 +1227,29 @@ def test_weblinks_when_description_is_not_in_the_kb():
...
+
+            return [MockRole(name) for name in self.user_roles]
+
+    superadmin_user = MockUser(user_roles=['superuser'])
+    cataloger_user = MockUser(user_roles=['cataloger'])
+    no_role_user = MockUser(user_roles=[])
+
+    assert is_cataloger(superadmin_user) == True
+    assert is_cataloger(cataloger_user) == True
+    assert is_cataloger(no_role_user) == False
```
Comment (jacquerie) [inline tests/unit/theme/test_theme_jinja2filters.py]: You can just:
```
assert is_cataloger(superadmin_user)
assert is_cataloger(cataloger_user)
assert not is_cataloger(no_role_user)
```

**Comment by jacquerie** [comment]
The only problem I see with this PR is that some catalogers are already using the tool, but none of them has an account, so we should give them one before deploying!

**Comment by kaplun** [comment]
In principle all catalogers have a local account on INSPIRE legacy. We should transfer their account (is legacy and Invenio 3 still using the same salting method)? Also on QA we can keep enabled local=1.

**Comment by jmartinm** [comment]
Yes, most of the catalogers already have a local account, and if not we can create new ones. Note that with the direct link also it will work as usual, this is just hiding the link from the dropdown.

>is legacy and Invenio 3 still using the same salting method

Have not tried in the past porting a legacy account into Invenio3 but they seem compatible checking the user migration task (https://github.com/inveniosoftware/invenio-migrator/blob/master/invenio_migrator/tasks/users.py#L68-L86)

**Pull request merged**